s4/str_case.rb

```diff
@@ -0,0 +1,36 @@
+# another case
+scase = "pRuEBa"
+p scase
+p scase.capitalize
+p scase.swapcase
+puts
+
+# .reverse
+p scase.capitalize.reverse
+puts
+
+# BANG methods
+p scase
+print "scase.capitalize: "; p scase.capitalize
+p scase
+print "scase.capitalize!: "; p scase.capitalize!
```

Comment:
What's the difference between `capitalize` and `capitalize!`?

Reply:
That the admiration sign assigns the return of the method to the variable. This works with several methods.